docs/faststore/components/features/delivery-promise/overview.mdx

```diff
@@ -33,32 +33,76 @@ Delivery Promise offers shoppers two product filtering options:
+Shoppers can filter available products globally using the **Filter by store** option, which displays only products available for pickup at a specific store. This filter shows all available pickup points within a 50-km radius, as configured in Checkout. The number of pickup points displayed is unlimited.
 
-![filter-by-store](https://vtexhelp.vtexassets.com/assets/docs/src/by-store___745e8c3ae10a84e55f88b107e5b00218.png)
+![filter-by-store](https://vtexhelp.vtexassets.com/assets/docs/src/dp-filter-by-store___6e22784e6d6a26719d8d7fbc6dfa0e43.gif)
 
 ### PLP or search page filters
 
-In PLPs or search pages, the shopper may have the following options to filter:
+On PLPs or search pages, the shopper has the following options for applying filters: [Delivery Methods](#delivery-methods-filters) and [Delivery Option](#delivery-option-filters).
 
-* **Deliver to:** Displays available products according to the shopper's location.
-* **Pickup at:** Displays the store set in the global pickup point filter, or the one closest to the shopper's location. To change the store set in `Pickup at {storeName}`, the shopper can click on it and select a new store in the modal that opens. This filter shows all available pickup points within a 50 km radius for pickup configured in Checkout. The number of pickup points displayed is unlimited.
-* **Pickup nearby:** Displays products available for pickup at nearby stores. This filter shows up to 40 pickup points within 10 km of the shopper's location.
-* **Pickup anywhere:** Displays products available for pickup at any participating store within the merchant's network, rather than restricting results to a selected pickup location or geographic proximity. **It's mainly used by B2B stores**.
+>⚠️ To edit the filter labels, follow the instructions in [Managing filters and badges](https://developers.vtex.com/docs/guides/faststore/delivery-promise-implementation).
 
-![plp-filters](https://vtexhelp.vtexassets.com/assets/docs/src/plp-by-shipping___e9ba4d4e3f0840c3ec53dc94ac7aafc4.png)
+#### Delivery Methods
+
+This filter displays available products according to the shopper's location. With this option, shoppers can filter using the following options:
+
+![dp-delivery-methods](https://vtexhelp.vtexassets.com/assets/docs/src/dp-filters-delivery-method___945ea013036a9ebebfc0078cb93796d9.png)
+
+| **Filter option** | **Description** |
+| ----------------- | --------------- |
+| **All delivery methods** | Displays products that match all active delivery method filters: **Shipping to**, **Pickup anywhere**, **Pickup at**, and **Pickup nearby**. |
+| **Shipping to**  |  Displays products that can be shipped to the shopper's specified location. |
+| **Pickup anywhere** | Displays products available for pickup at any participating store within the merchant's network, rather than restricting results to a selected pickup location or geographic proximity. This is often used by B2B stores. |

```

Comment:
```suggestion
| **Pickup anywhere** | Displays products available for pickup at any participating store within the merchant's network, rather than restricting results to a selected pickup location or geographic proximity. This is often used by B2B stores and this filter is deactivated in hCMS by default. |

```